src/octoprint/printer/standard.py

```diff
@@ -927,6 +927,7 @@ def is_ready(self, *args, **kwargs):
         return (
             self.is_operational()
             and not self.is_printing()
+            and not self.is_paused()
             and not self._comm.isStreaming()
         )
```

Comment:
Good find! However, in order to not duplicate logic here, I'd rather go with:

```suggestion
        return (
            self.is_operational()
            and not self._comm.isBusy()
            and not self._comm.isStreaming()
        )
```

Do you want to do the honors or shall I?

Reply:
I did, and added a comment since comm being busy is not obvious that it includes paused.